psvpfsparser/FilesDbParser.cpp

```diff
@@ -265,12 +265,6 @@ bool FilesDbParser::validate_header(uint32_t dataSize)
       return false;
    }
 
-   if(m_header.unk6 != 0xFFFFFFFFFFFFFFFF)
```

Comment:
Please - do not remove this check.
If it can contain other values, as you have indicated in the comments - include those values in the check.
This way we can build up statistics on possible values and maybe understand what this field means.

Reply:
added this back

https://github.com/motoharu-gosuto/psvpfstools/blob/94471aefbc4e991bc2981f1489237df9c7ff44f2/psvpfsparser/FilesDbParser.cpp#L268-L272

Reply:
Perfect. This is very interesting that unk6 equals 0x400. Which is EXPECTED_BLOCK_SIZE in files.db. I wonder if this is a coincidence.